examples/backend-service/main.tf

```diff
@@ -113,34 +79,27 @@ resource "aws_service_discovery_service" "sd_service" {
 module "ecs_service_definition" {
   source = "../../modules/ecs-service"
 
-  name                       = var.service_name
-  desired_count              = var.desired_count
-  ecs_cluster_id             = data.aws_ecs_cluster.core_infra.cluster_name
-  cp_strategy_base           = var.cp_strategy_base
-  cp_strategy_fg_weight      = var.cp_strategy_fg_weight
-  cp_strategy_fg_spot_weight = var.cp_strategy_fg_spot_weight
+  name           = local.name
+  desired_count  = 3
```

Comment:
Is it not good ta be able to configure the number with a var or a local ?

Reply:
- module: A container for multiple resources that are used together to create lightweight abstractions. A module accepts inputs to configure and influence the resulting resources, and provides zero or more outputs. Modules are generally sourced from the Terraform registry but can also be sourced from a local directory, from GitHub, or a private registry.

- blueprint/example: An example implementation of one or more modules and/or one one or more resource objects that demonstrates an outcome. A blueprint/example is not intended to be used in-place nor is it intended to be used as a source like that found within a module. To use a blueprint/example, users can copy+paste the relevant code into their own project, or simply use a reference for how to structure their own code/project